doc/classes/Vector2.xml

```diff
@@ -303,7 +305,7 @@
 			<argument index="0" name="n" type="Vector2">
 			</argument>
 			<description>
-				Returns the component of the vector along a plane defined by the given normal.
+				Returns this vector slid along a plane defined by the given normal.
```

Comment:
I'm not sure this is clear enough as is (wasn't clear enough before either), especially to non-native speakers who may not be familiar with the past-participle of "slide" (or for anyone, the meaning of "sliding a vector along a plane").

Reply:
I'm not really familiar with this function; I don't know what it's used for. This is the best documentation I can write for it, perhaps someone else can improve it further.